tests/test_PessimisticPatternMatcherB3D.py

```diff
@@ -0,0 +1,273 @@
+
```

Comment:
In the future Russell, could you please be more clear as to what you mean. I haven't been working in LSST long enough to know what the usual boilerplate means.